src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcPageHeadingSection.scala.html

```diff
@@ -0,0 +1,20 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
```

Comment:
- [ ] create a ticket to find out why the headers aren't being added to sbt files because I did not manage to work it out quickly and decided to defer it

Reply:
https://jira.tools.tax.service.gov.uk/browse/PLATUI-1326